### PR TITLE
Allow user to resend account activation link

### LINKED ISSUE #212: Add a way to re-send confirmation link
If users register on LibreCores they are sent an email with a link to confirm their email address. The UI should provide a way to re-send this confirmation link.

### PR DESCRIPTION
**Pull request by agathver**
Fixes #212

### CODE REVIEW

File: site/src/Librecores/ProjectRepoBundle/Controller/UserController.php [hunk 1]
```diff
@@ -177,4 +181,43 @@ public function passwordSettingsAction(Request $request, UserManagerInterface $u
...
         );
     }
+
+    /**
+     * @param Request $request
+     *
+     * @return RedirectResponse|Response
+     */
+    public function resendConfirmationEmailAction(Request $request) {
+        // TODO: Use auto-wired dependencies; don't fetch from the container
```
[the remark this answers is not in the document]
Comment (agathver) [reply site/src/Librecores/ProjectRepoBundle/Controller/UserController.php]: Autowiring configuration landed in #312 

I'll rebase and remove such TODO

**Comment by imphil** [comment]
Looks like this needs a rebase on top of master to get the phpcs.xml changes out of this PR?

**Comment by imphil** [comment]
I tried this now and it doesn't work for me.

STR:
- Enable user email verification
- Register a new account
- Try to log in with this new account.
- The login fails with the message "Your need to activate your account before you can login. Follow the instructions sent to your registered email address. To resend your confirmation email click here."
- Clicking on the provided link gets me to http://www.librecores.devel/user/resend-confirmation

Result:
This page shows:

```
RuntimeException:
Controller "Librecores\ProjectRepoBundle\Controller\UserController::resendConfirmationEmailAction()" requires that you provide a value for the "$mailer" argument. Either the argument is nullable and no null value has been provided, no default value has been provided or because there is a non optional argument after this one.

  at vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Controller/ArgumentResolver.php:78
  at Symfony\Component\HttpKernel\Controller\ArgumentResolver->getArguments(object(Request), array(object(UserController), 'resendConfirmationEmailAction'))
     (vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Controller/TraceableArgumentResolver.php:38)
  at Symfony\Component\HttpKernel\Controller\TraceableArgumentResolver->getArguments(object(Request), array(object(UserController), 'resendConfirmationEmailAction'))
     (vendor/symfony/symfony/src/Symfony/Component/HttpKernel/HttpKernel.php:143)
  at Symfony\Component\HttpKernel\HttpKernel->handleRaw(object(Request), 1)
     (vendor/symfony/symfony/src/Symfony/Component/HttpKernel/HttpKernel.php:68)
  at Symfony\Component\HttpKernel\HttpKernel->handle(object(Request), 1, true)
     (vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Kernel.php:200)
  at Symfony\Component\HttpKernel\Kernel->handle(object(Request))
     (web/app_dev.php:26)
```

Expected result:
something more useful

**Comment by imphil** [comment]
In addition to fixing the exception above (which still happens after a rebase on top of `master`), please add the resend email link to the page http://www.librecores.devel/user/register/check-email which is shown after the user registers.

**Pull request merged**